buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/Spotless.kt

```diff
@@ -22,6 +22,7 @@ fun Project.configureSpotless() {
     }
 
     val ktlintEditorConfig = mapOf(
+      "ktlint_standard_comment-wrapping" to "disabled",
```

Comment:
![image](https://github.com/VirtusLab/git-machete-intellij-plugin/assets/19799111/212c50af-c0e7-4b37-81e1-8ec7875d327c)

Reply:
Every update of ktlint must change something :trollface: